src/DataTable/EmptyTable.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const EmptyTable = () => <div className="pgn__table-empty">No results</div>;
```

Comment:
We discussed renaming this to `EmptyTableContent` or something similar right?

Reply:
Yessss but I forgot